Purchases/Misc/RCLogUtils.h

```diff
@@ -1,5 +1,5 @@
 //
-//  RCUtils.h
+//  RCLogUtils.h
```

Comment:
I meant to rename this class in #237 but I forgot. It only does logging stuff now